lib/ansible/module_utils/basic.py

```diff
@@ -2010,9 +2010,6 @@ def _return_formatted(self, kwargs):
     def exit_json(self, **kwargs):
         ''' return from the module, without error '''
 
-        if not 'changed' in kwargs:
-            kwargs['changed'] = False
-
```

Comment:
not sure if we should remove this, it is still nice for modules to return `changed=False`, the task executor side just ensures that even 'non basic.py using modules' also do